### PR TITLE
[4870] Employing School missing - pls add to register

### PR DESCRIPTION
**Pull request by d-a-v-e**
### Context

A new school on GIAS is missing from Register stopping an Employment School from being provided. The school is called Harris Science Academy East London, the urn is 149431, however, we should import to the schools table the new list from GIAS, rather than add this 1 school individually.

### Changes proposed in this pull request

new rake task for GIAS schools data

### Guidance to review

* run locally and see if the rake task errors `be rake schools_data:import_gias`
* I've also removed the CSV for disability data from PR #2827 

### Important business

- [ ] ~Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?~
- [ ] ~Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?~

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: db/data/20221011140515_add_missing_disabilities_from_tad_check.rb [hunk 1]
```diff
@@ -72,6 +76,8 @@ def other_disability
   end
 
   def csv
-    @csv ||= CSV.table(Rails.root.join("data/tad_disability_check_2022-10-03.csv"))
+    return unless Rails.root.join(FILE).exist?
+
+    @csv ||= CSV.table(Rails.root.join(FILE))
```
Comment (d-a-v-e) [inline db/data/20221011140515_add_missing_disabilities_from_tad_check.rb:81]: This is unrelated to the context of this PR, but removes a redundant CSV (see guidance for review)

**Comment by d-a-v-e** [comment]
(I'll squash the PR to get rid of those erroneous commits)

**Comment by d-a-v-e** [comment]
[Employing School missing - pls add to register](https://trello.com/c/BaiQti3d/4870-employing-school-missing-pls-add-to-register)

**Review by jhenderson — APPROVED** [review]
Tried it out locally. LGTM.

**Comment by claire-hughez** [comment]
Checked in production data. I checked all our existing Lead Schools were marked as Lead Schools and the new one was added. Looks good, thank you @d-a-v-e

**Pull request merged**